lucid-book-store/src/components/cart-wishlist/cart-card.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const CartCard = () => {
+  return (
+    <>
```

Comment:
no need to give an extra div since everything below is already wrapped in a div

Reply:
Yes, made the changes